modules/Router.js

```diff
@@ -62,6 +62,8 @@ const Router = React.createClass({
       if (error) {
         this.handleError(error)
       } else {
+        router.location = state.location
+        router.params = state.params
```

Comment:
Would adding `routes` here make sense as well? I've been trying to add tracking to an app with redux & react-router, in some redux middleware I imagine seeing me use this like:

```js
import { LOCATION_CHANGE } from 'react-router-redux';

export default function analytics({ getState }) {
  return (dispatch) => (action) => {
    switch (action.type) {
      case LOCATION_CHANGE:
        track({ name: 'visit', ...visitTrackOptions(action.payload) });
        break;
    }

    return dispatch(action);
  };
}

function visitTrackOptions({ location, routes }) {
  const routePaths = routes.map(route => route.path);
  const pattern = filter(routePaths).join('/').replace('//', '/');

  return {
    weblink_url_full: location.pathname, //=> `/posts/foo-bar`
    weblink_url_pattern: pattern, //=> `/posts/:slug`
  };
}
```
Is adding `routes` here something that can be considered? Reason why I ask is because we've been trying to add tracking to an app via redux & react-router